requirements.txt

```diff
@@ -1,7 +0,0 @@
-affine>=1.3.0
-boto3>=1.2.4
-cligj
-enum34
-numpy>=1.8
-snuggs>=1.2
-setuptools
```

Comment:
I understand the goal to reduce duplication but the `requirements*.txt` files are required for certain workflows. They serve a slightly different purpose than setup.py: https://caremad.io/2013/07/setup-vs-requirement/ 

I'd prefer to leave them intact and accept a bit of duplication.